spec/models/service_spec.rb

```diff
@@ -1,5 +1,15 @@
 require 'rails_helper'
 
 RSpec.describe Service, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  context 'database associations' do
+    it { is_expected.to have_many(:items) }
+  end
+
+  context 'model validations' do
+    it { is_expected.to validate_presence_of(:name) }
+  end
+
+  context 'model attributes' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.